### ISSUE TITLE
Remove NES controller documentation

### ISSUE THREAD

**Issue by mist64** [commanderx16/x16-docs#69]
During development, it was not decided whether the final hardware would have NES or SNES controller ports. The decision has now been made in favor of SNES controllers, so all references to NES controllers and the bit layout in their data word needs to be removed.

**Comment by kkarhan** [comment]
Yeah, it should reference the fact that it uses SNES controllers due to being an easier to handle interface and supporting more buttons [10 instead of 6].

Personally, I would've chosen Neo Geo Controller Ports as they electromechanically can't fit 9-pin Genesis/Megadrive or Atari/Commodore Joysticks, but that would've meant needing to handle a 15-pin connector at not much of a benefit and increased cost.
